docs/tutorial/anonlink-novt.ipynb

```diff
@@ -547,19 +575,28 @@
     "!anonlink describe 'novt_clk_0.json'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case, the popcount mean is not very large compared to bloom filter length (1024). If the popcount mean is large, you can reduce it by modifying the schema. For more details, please have a look at this <a href='https://clkhash.readthedocs.io/en/stable/tutorial_cli.html'>tutorial</a>. "
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### Block dataset\n",
     "\n",
     "\n",
-    "Blocking is a technique that makes record linkage scalable. It is achieved by partitioning datasets into groups, called blocks and only comparing records in corresponding blocks. This can reduce the number of comparisons that need to be conducted to find which pairs of records should be linked."
+    "Blocking is a technique that makes record linkage scalable. It is achieved by partitioning datasets into groups, called blocks and only comparing records in corresponding blocks. This can reduce the number of comparisons that need to be conducted to find which pairs of records should be linked.\n",
+    "\n",
+    "Similar to the hashing above, the blocking is configured with a schema. For this linkage we chose 'lambda-fold' as blocking technique. This blocking method is proposed in paper *An LSH-Based Blocking Approach with a Homomorphic Matching Technique for Privacy-Preserving Record Linkage*. We also provide a detailed explanation about how this blocking method works in this <a href='https://github.com/data61/blocklib/blob/master/docs/tutorial/tutorial_blocking.ipynb'>tutorial</a>."
```

Comment:
add the link to the paper.

We also provide a detailed explanation ~~about~~ **of** how this ...